catalog/CatalogDatabase.hpp

```diff
@@ -290,7 +290,7 @@ class CatalogDatabase : public CatalogDatabaseLite {
    * @param id The id to search for.
    * @return The relation with the given ID.
    **/
-  const CatalogRelation* getRelationById(const relation_id id) const {
+  const CatalogRelation* getRelationById(const relation_id id) const override {
```

Comment:
Per [the comment](https://github.com/pivotalsoftware/quickstep/pull/238/files#r64515758), please reset this change.